### PR TITLE
[CH] Migrate get_workflow_jobs

### PR DESCRIPTION
**Pull request by huydhn**
An easy one, this query is only used to fetch the log link on compiler benchmark page

### Testing

https://torchci-git-fork-huydhn-ch-migrate-getworkflowjobs-fbopensource.vercel.app/benchmark/compilers

### CODE REVIEW

File: torchci/components/benchmark/compilers/BenchmarkLogs.tsx [hunk 1]
```diff
@@ -2,26 +2,21 @@ import { LOG_PREFIX } from "components/benchmark/common";
...
-    {
-      name: "workflowId",
-      type: "int",
-      value: workflowId,
-    },
-  ];
-  const url = `/api/query/${queryCollection}/${queryName}?parameters=${encodeURIComponent(
+  const queryParams: { [key: string]: any } = {
+    jobName: "%test (%",
+    workflowId: workflowId,
```
Comment (clee2000) [inline torchci/components/benchmark/compilers/BenchmarkLogs.tsx:17]: Does str/int matter here?

**Review by clee2000 — APPROVED** [review]

File: torchci/components/benchmark/compilers/BenchmarkLogs.tsx (hunk 1 above)
Comment (huydhn) [reply torchci/components/benchmark/compilers/BenchmarkLogs.tsx:17]: The preview page looks fine, so I think there is no difference

Comment (huydhn) [reply torchci/components/benchmark/compilers/BenchmarkLogs.tsx:17]: Screenshot 2024-10-24 at 17 36 01

This part shows up ok on https://torchci-git-fork-huydhn-ch-migrate-getworkflowjobs-fbopensource.vercel.app/benchmark/compilers

Comment (huydhn) [reply torchci/components/benchmark/compilers/BenchmarkLogs.tsx:17]: I switch the type of `workflowId` to a number anyway as it looks better

**Pull request merged**